docs/reference/cluster/stats.asciidoc

```diff
@@ -188,8 +193,19 @@ Will return, for example:
           "classname": "org.elasticsearch.ingest.useragent.IngestUserAgentPlugin",
           "has_native_controller": false
         }
-      ]
+      ],
+      "network_types" : {
+         "transport_types" : {
+            "netty4" : 1
+         },
+         "http_types" : {
+            "netty4" : 1
+         }
+      }
    }
 }
 --------------------------------------------------
+// TESTRESPONSE[s/"plugins": \[[^\]]*\]/"plugins": $body.$_path/]
+// TESTRESPONSE[s/: (\-)?[0-9]+/: $body.$_path/]
+// TESTRESPONSE[s/: "[^"]*"/: $body.$_path/]
```

Comment:
This replace all the fields values by the expected ones in the test. We don't care about the field values, but we need to test the field names.